VoxeUnity/Assets/Voxelmetric/Code/Core/Chunk.cs

```diff
@@ -52,9 +52,9 @@ public partial class Chunk : ChunkEventSource
         //! List of neighbors
         public Chunk[] Neighbors { get; } = Helpers.CreateArray1D<Chunk>(6);
         //! Current number of neigbors        
-        public int NeighborCount { get; private set; }
+        public byte NeighborCount { get; private set; }
         //! Maximum possible number of neighbors given the circumstances
-        public int NeighborCountMax { get; private set; }
+        public byte NeighborCountMax { get; private set; }
```

Comment:
While this might be an optimization of sorts, I don't believe it's necessary at this point. Even with 10.000 chunks, we're talking about saving ~60 kiB of memory here.